### DISCUSSION TITLE
Hi Showtime Team 👋

### DISCUSSION THREAD

**Discussion by marklawlor** [showtime-xyz/showtime-frontend#1127] [General]
Hi Showtime Team 👋

I thought I'd introduce myself before I start appearing in PRs. My name is Mark, I'm the creator of tailwindcss-react-native, and I'm super excited to help you guys transition your components to my library. After speaking with @axeldelafosse it's great to meet a team that has the same vision of simple, universal applications.

The library still quite new and the documentation is...getting there :man_shrugging:  So if you have any questions about the library or even suggestions please reach out. 

I'm based in Australia, which has the worst timezone crossover with the rest of the world, so my replies might be a bit delayed.

Happy to meet you all :smile:

**Comment by axeldelafosse** [comment]
Hey Mark! So happy to see you here!

> it's great to meet a team that has the same vision of simple, universal applications

✨

**Comment by intergalacticspacehighway** [comment]
Hi Mark! 👋  Happy to meet you as well! 😄

**Comment by enesozturk** [comment]
Welcome, Mark! Glad to see you here 🤞🏽

**Comment by alantoa** [comment]
Hey Mark @marklawlor, nice to meet you!!! 😆  
I'm very like your library!
